### ISSUE TITLE
Unhandled promise rejection: TypeError: this.reject is not a function

### ISSUE THREAD

**Issue by FDiskas** [parcel-bundler/parcel#1050]
On a bigger project got the folowing error then parcel is running.
Nothing special, just allot react components in a project. Used scss and css modules
```
Unhandled promise rejection: TypeError: this.reject is not a function
    at PromiseQueue._runJob (/mnt/SSD/Projects/***/node_modules/parcel-bundler/src/utils/PromiseQueue.js:50:12)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Unhandled promise rejection: TypeError: this.reject is not a function
    at PromiseQueue._runJob (/mnt/SSD/Projects/***/node_modules/parcel-bundler/src/utils/PromiseQueue.js:50:12)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Unhandled promise rejection: TypeError: this.reject is not a function
    at PromiseQueue._runJob (/mnt/SSD/Projects/***/node_modules/parcel-bundler/src/utils/PromiseQueue.js:50:12)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```
parcel: 1.6.2

**Comment by FDiskas** [comment]
Usual it happens then I install additional npm module to project while parcel is running :pray:

**Comment by FDiskas** [comment]
Can you suggest me how todo so?

**Comment by mischnic** [comment]
in your package.json:
```json
{
  "devDependencies": {
    "parcel-bundler": "parcel-bundler/parcel"
  }
}

```

Then reinstall your dependencies.

**Comment by gnijuohz** [comment]
@FDiskas You can clone this project and build it yourself by running: `git clone git@github.com:parcel-bundler/parcel.git && cd parcel && yarn install && yarn build` and then you can use `yarn link` as described in the second option [here](git@github.com:parcel-bundler/parcel.git)

**Comment by FDiskas** [comment]
Looks like it's fixed. 
Just keep in mind feels like project build time is 2x slower :stuck_out_tongue_closed_eyes:

**Comment by fathyb** [comment]
@FDiskas it should be slower the first time because we are fetching dev dependencies, the next build (even without cache) should be sustainably faster.

@davidnagli any idea for the `uglify-es` error above?

**Comment by davidnagli** [comment]
@fathyb No clue

@FDiskas Can you please provide an example repo so I can try to reproduce?

**Comment by FDiskas** [comment]
Sorry repo is private and it's not easy reproducible. I will create another issue if it will popup next time. :smile:

**Comment by fathyb** [comment]
Fixed in `1.7.0`, feel free to reopen if needed.
